fyde-app/README.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Fyde App
+has_children: false
+nav_order: 3
+permalink: /fyde-app
+---
+# Fyde App
+{: .no_toc }
+
+The Fyde app bundles the end user UI and the Fyde agent. It is available on the following platforms:
+
+* iOS - available on the [App Store](https://www.fyde.com/downloads)
```

Comment:
@luisalima there are platform specific links.

https://www.fyde.com/downloads/ios
https://www.fyde.com/downloads/linux
https://www.fyde.com/downloads/macos
https://www.fyde.com/downloads/windows

You're also missing Android
https://www.fyde.com/downloads/android

Reply:
Updated